database-seed/mongo-loader.js

```diff
@@ -19,14 +19,37 @@ const productImageScheme = mongoose.Schema({
 let Product = mongoose.model('ProductImage', productImageScheme);
 
 const seedDatabase = () => {
-    let documents = JSON.parse(retrieveData());
-    documents.forEach((document) => {
-        Product.updateOne({'productID':document.productID}, document, {upsert: true}, (err) => {
-            if (err) throw err;
+    dropDatabase()
+    .then(() => {
+        let documents = JSON.parse(retrieveData());
+        let updates = [];
+        documents.forEach((document) => {
+            updates.push(update(document));
+        });
+        Promise.all(updates).then(() => {
+            console.log('Database seeded');
+            process.exit();
         })
     });
-    console.log('seed complete, exiting in 5 seconds');
-    setTimeout(()=>{process.exit()}, 5000);
+}
+
+const update = (document) => {
+    return new Promise((resolve, reject) => {
+        Product.updateOne({'productID':document.productID}, document, {upsert: true}, (err) => {
+            if (err) reject(err);
+            resolve();
+        });
+    });
+}
+
+const dropDatabase = () => {
+    return new Promise((resolve, reject) => {
+        Product.deleteMany({}, (err) => {
```

Comment:
Does the delete many function delete the previous database?

Reply:
It queries the database with the query (in this case `{}`), and then deletes all responses.  By querying with `{}`, it receives all results.